### PR TITLE
during installation inform user that pwd cannot be blank

### PR DESCRIPTION
**Pull request by tobiasd**
Inform the users during the installation that the DB passwd cannot be blank, see original discussion on Friendica, relates to #3114

### CODE REVIEW

File: mod/install.php [hunk 1]
```diff
@@ -261,7 +261,7 @@ function check_passed($v, $c){
 
 				'$dbhost' => array('dbhost', t('Database Server Name'), $dbhost, '', 'required'),
 				'$dbuser' => array('dbuser', t('Database Login Name'), $dbuser, '', 'required', 'autofocus'),
-				'$dbpass' => array('dbpass', t('Database Login Password'), $dbpass, '', 'required'),
+				'$dbpass' => array('dbpass', t('Database Login Password'), $dbpass, t('For security reasons this cannot be empty'), 'required'),
 				'$dbdata' => array('dbdata', t('Database Name'), $dbdata, '', 'required'),
```
Comment (annando) [inline mod/install.php]: I would prefer "For security reasons the password mustn't be empty".

**Comment by annando** [comment]
This is a good quick fix. We should - however - add a check later.

**Comment by tobiasd** [comment]
What kind of check? The form field is required, any modern browser will not go on unless the field is filled out.

File: mod/install.php [hunk 2]
```diff
@@ -261,7 +261,7 @@ function check_passed($v, $c){
 
 				'$dbhost' => array('dbhost', t('Database Server Name'), $dbhost, '', 'required'),
 				'$dbuser' => array('dbuser', t('Database Login Name'), $dbuser, '', 'required', 'autofocus'),
-				'$dbpass' => array('dbpass', t('Database Login Password'), $dbpass, t('For security reasons this cannot be empty'), 'required'),
+				'$dbpass' => array('dbpass', t('Database Login Password'), $dbpass, t("For security reasons thr password must not be empty"), 'required'),
```
Comment (annando) [inline mod/install.php]: "the" instead of "thr" would even be more better :-)

**Comment by annando** [comment]
I meant a check if the database connection can be established at all, not only because of an empty password but also because of a wrong hostname, wrong password, ...

File: mod/install.php (hunk 2 above)
Comment (tobiasd) [inline mod/install.php]: I'm just gaming the commit count statistics.

**Comment by tobiasd** [comment]
Ah ok. Yes that check should be there and give feedback to the user.

**Comment by tobiasd** [comment]
But I wont do the check in this PR ;-)

**Pull request merged**